app/views/api/v1/change_requests/show.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+json.extract! @description_change_request,
+              :id,
+              :planning_application_id,
+              :state,
+              :response_due,
+              :proposed_description,
+              :rejection_reason,
+              :approved
```

Comment:
On further thought, I really don't think we need this action to exist, at least not yet. If we do want it to exist, then we should be writing a `spec/requests/api/change_request_show_spec.rb`, but I think this is us writing and maintaining an active API endpoint that we don't have a need for yet.